### PR TITLE
Add download page for 22.02 [skip ci]

### PR DESCRIPTION
**Pull request by viadea**
Add download page for 22.02

Signed-off-by: Hao Zhu <hazhu@nvidia.com>

### CODE REVIEW

**Review by jlowe — COMMENTED** [review]
This seems a bit premature.  There are pending PRs, like the mixed join support in #4477 and potentially #4531, that we may want to highlight on the download page for this release.  Also I don't think we would want to merge this until just before releasing so we don't have dangling links in our documentation longer than necessary.

**Comment by viadea** [comment]
@jlowe we are preparing the doc updates PRs in advance this time as per @sameerz .
Especially for download page PR, it would be nice we can collect the feedback from you and the team to add anything missing. 
This initial version is based on PRD doc, so it might not be fully complete.

**Comment by jlowe** [comment]
I'm fine with iterating here on the PR, but I don't think we should merge this until relatively late.  Marking this as a draft until we're ready so it doesn't get accidentally merged early if someone swings by and notices it's approved.

**Comment by viadea** [comment]
> I'm fine with iterating here on the PR, but I don't think we should merge this until relatively late. Marking this as a draft until we're ready so it doesn't get accidentally merged early if someone swings by and notices it's approved.

Cool. I just added a line for mixed join feature:
```
* Support LeftOuter, RightOuter, FullOuter, Inner, hash semi and anti joins using libcudf's mixed join feature
```

Feel free to re-word if it is not accurate.

File: docs/download.md [hunk 1]
```diff
@@ -18,6 +18,69 @@ cuDF jar, that is either preinstalled in the Spark classpath on all nodes or sub
...
+* MIG on YARN support
+* GPU explain only mode for Spark 3.x and 2.x
+* JSON reader support
+* Sequence function support
+* regexp_extract function support
+* Min and max on single-level struct
+* CreateMap updates and enable CreateMap by default
+* Cast from array to string
+* Add regular expression support to regexp_replace function
+* Support LeftOuter, RightOuter, FullOuter, Inner, hash semi and anti joins using libcudf's mixed join feature
```
Comment (jlowe) [inline docs/download.md:79]:
```suggestion
* Support for conditional joins using libcudf's mixed join feature
```

File: docs/download.md [hunk 2]
```diff
@@ -18,6 +18,69 @@ cuDF jar, that is either preinstalled in the Spark classpath on all nodes or sub
...
+
+	GPU Models: NVIDIA V100, T4 and A2/A10/A30/A100 GPUs
+
+Software Requirements:
+
+	OS: Ubuntu 18.04, Ubuntu 20.04 or CentOS 7, CentOS 8
+
+	CUDA & NVIDIA Drivers*: 11.x & v450.80.02+
+
+	Apache Spark 3.0.1, 3.0.2, 3.0.3, 3.1.1, 3.1.2, 3.2.0, Cloudera CDP 7.1.6, 7.1.7, Databricks 7.3 ML LTS or 9.1 ML LTS Runtime and GCP Dataproc 2.0
```
Comment (sameerz) [inline docs/download.md:34]: We are going to try to support 3.2.1 in the upcoming release, assuming the artifact is available on time.  We may need to update this before merging the PR.

Comment (viadea) [reply docs/download.md:34]: Sure @sameerz . If 3.2.1 is arriving on time, i will add it here.

Comment (viadea) [reply docs/download.md]: Added 3.2.1

**Review by jlowe — APPROVED** [review]

**Comment by viadea** [comment]
build

**Comment by jlowe** [comment]
build

**Pull request merged**